apps/desktop/src/hooks/use-platform-info.ts

```diff
@@ -11,33 +11,37 @@ export const usePlatformInfo = () => {
     os: "",
     kernalVersion: "",
     arch: "",
+    name: "",
     configDir: "",
   });
 
   useEffect(() => {
-    const allPromises = [getTauriVersion(), getVersion(), getPlatform(), getKernalVersion(), getArch(), appConfigDir()];
+    const allPromises = [getTauriVersion(), getVersion(), getPlatform(), getKernalVersion(), getArch(), appConfigDir(), getName()];
 
     // get all the dataz
     Promise.allSettled(allPromises).then(results => {
-      const [tauriVersion = "", appVersion = "", os = "", kernalVersion = "", arch = "", configDir = ""] = results.map(
+      const [tauriVersion = "", appVersion = "", os = "", kernalVersion = "", arch = "", configDir = "", name = ""] = results.map(
         result => {
           if (result.status === "fulfilled") {
             return result.value;
           }
           return "";
         }
       );
-
       setPlatformInfo({
         tauriVersion,
         appVersion,
         os,
         kernalVersion,
         arch,
         configDir,
+        name
       });
     });
   }, []);
 
   return platformInfo;
 };
+export const useCanary = () => {
+  return usePlatformInfo().name.includes("Canary");
```

Comment:
This looks like it should work fine!
![image](https://github.com/Hacksore/overlayed/assets/996134/5fe6a14c-fb18-4854-8c74-0fcaa68226cd)